Hai secret no more (#4952)
endless-sky/endless-sky: data/hai/hai reveal 5 philosophers.txt

# Copyright (c) 2020 Nomadic Volcano
#
# Endless Sky is free software: you can redistribute it and/or modify it under the
# terms of the GNU General Public License as published by the Free Software
# Foundation, either version 3 of the License, or (at your option) any later version.
#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A

> # PARTICULAR PURPOSE.  See the GNU General Public License for more details.

Reviewer: ```suggestion
# PARTICULAR PURPOSE. See the GNU General Public License for more details.
#
# You should have received a copy of the GNU General Public License along with
# this program. If not, see <https://www.gnu.org/licenses/>.
```

			`	"Hai are all banned from entering human space now, and humans are not allowed in and out except for critical travel. My new position as an Elder grants me special privileges. Please, in the name of peace, escort my ship to meet your authorities. I must also see Waypoint and Ultima Thule for myself. I dare not try this without your escort. You will be well-paid."`
			choice
				`	"Of course. Let's go."`
					accept
				`	"Not yet. I need to make some preparations first."`
					decline
	on complete
		payment 400000
	npc save accompany
		government "Hai (Wormhole Access)"
		personality timid escort
		ship "Centipede (Teeneep)" "Teeneep's Ship"

mission "Hai Rescue [B02B] Diplomacy: Syndicate (Retry)"
	landing
	name "Missing Researchers"
	description "Visit <waypoints>, then escort Teeneep's Centipede to <destination>. Payment is <payment>."
	source "Hai-home"
	waypoint "Waypoint"
	waypoint "Ultima Thule"
	destination "Sunracer"
	repeat
	to offer
		has "Hai Rescue [B02] Diplomacy: Syndicate: declined"
		not "Hai Rescue [B02B] Diplomacy: Syndicate (Retry): failed"
		not "Hai Rescue [B02B] Diplomacy: Syndicate (Retry): done"

> 		not "Hai Rescue [B02B] Diplomacy: Syndicate (Retry): active"

Author: ![image](https://user-images.githubusercontent.com/60949828/192076014-f1b0dd56-e97d-4fef-a9ff-f7c7cebedf82.png)

		conversation
			`Earth authorities ask you to land in the Republic Capital Building, a vast skyscraper over a kilometer high which has its own spaceport. You land in one of several hangars built for ships significantly larger than a Carrier. In the distance, you spy an unfamiliar ship design which looks like a massive gunship. You've heard rumors that the Navy is working on advanced ships built to fight alien races. They may have left one here for Teeneep to see, as a warning to the Hai Elders.`
			`	A formal delegation meets Teeneep at her ship, but she awaits you before continuing into the building. It takes about ten minutes of elevators and hallways to get to the top of the building: the Parliament Chamber. The chamber is empty; your delegation walks through it to a room on the far end of the chamber.`
			choice
				`	(Enter the room.)`
					goto room
				`	(Admire the scenery.)`

> 			`	The entire outer wall of the 200-meter-wide room is a transparent metal which lets you see the steel forest of buildings that coat Earth. You're at the top of the tallest structure ever built by the human race.  The high altitude of the Parliament Chamber symbolizes its status in society: the supreme elected authority that oversees billions of people.`

Reviewer 2: ```suggestion
			`	The entire outer wall of the 200-meter-wide room is a transparent metal which lets you see the steel forest of buildings that coat Earth. You're at the top of the tallest structure ever built by the human race. The high altitude of the Parliament Chamber symbolizes its status in society: the supreme elected authority that oversees billions of people.`
```

> 			`	Beyond the chamber resides the Crystal Room, perhaps the most famous room in the galaxy. It was there that monumental treaties were signed: the end of the Alpha Wars, the formal declaration of the illegality of slavery, the Syndicate and the Deep joining the Republic, the peace treaty with the Free Worlds, and numerous others.`

Reviewer 2: This building almost certainly did not exist all that time ago. And, if it did, I have trouble believing that it is still the tallest humans have built.

			`	Teeneep looks back at you, waiting for you to rejoin her.`
			choice
				`	(Enter the room.)`
			label room
			`	You're surprised to see before you the President of the Republic and the Chancellor of the Free Worlds.`

> 			`	Teeneep begins, "I speak as the voice of the Hai and with the full authority of the Elder Council. Many of our people have been kidnapped over the past few decades, and some still reside beyond the wormhole at Devil-Run. Worst of all, your authorities have killed numerous Hai on slaver ships, without attempting rescue or even offering surrender. We demand the return of our people and an immediate end to the killing of slaves."`

Reviewer 2: ```suggestion
			`	Teeneep begins, "I speak as the voice of the Hai and with the full authority of the Elder Council. Many of our people have been kidnapped over the past few decades, and some still reside beyond the wormhole at Devil-Run. Worst of all, your authorities have killed numerous Hai on slaver ships, without attempting rescue or even offering the opportunity to surrender. We demand the return of our people and an immediate end to the killing of slaves."`
```

			`	"The Free Worlds will also extend our protection, of course," the Chancellor adds.`
			`	Teeneep responds, "I cannot sign that treaty yet. My people won't be satisfied until the kidnapped Hai are returned."`
			`	The President replies, "We heard back from Syndicate Internal Affairs that they know the exact location of three staging facilities for a group called the 'Devil-Run Gang.' That group is behind the kidnappings." Turning to you, the President continues, "Please return to <destination>, and speak with Deputy Director Moray again. He may have a solution."`
			`	Teeneep concludes the discussion. "I will travel with Captain <last> and see this matter through. I will expect your fleets and armies to be ready to assist us when the time is right." She turns to leave.`
			choice
				`	(Return to your ship.)`

> 			`	On the way out, Teeneep tells you her next move. "The humans are so eager to raid the facilities that they haven't considered the need for information. We will have to arrange for someone to be 'captured' and brought to Philosopher's Paradise. The human authorities must delay their strike until then.`

Reviewer 2: When did we start calling it the "Philosopher's Paradise"?

			`	"I'll need an escort back to <destination>. We'll need help from one more group, but I'll contact them on the way. You'll be paid, of course." She hands you <payment> for her most recent journey.`
				accept
	on complete
		payment 400000
		conversation
			`You arrive at an office building with unusually high ceilings. When you enter, you notice one of the Quarg scientists you spoke to before is present, their height explaining the chosen meeting location. This must be the "other group" Teeneep said would help. You and Teeneep join the Quarg and Stephen Moray at a tall table.`
			`	Teeneep explains her "intentional kidnapping" plan to the others, and Stephen says, "Believe it or not, I met a Hai recently that might be able to help. His name is Tutitot Eeegin, and he's expressed interest in helping us solve this problem. He's clever, and he has some experience acting." Stephen excuses himself to contact Tutitot while the Quarg turns to you and Teeneep.`

> 			`	The Quarg in question begins in a low voice. "Teeneep here has asked if the Quarg have something you could use that would let Tutitot send telepathic signals millions of your kilometers. Alas, the Quarg have been unable to create such technology, and neither have the Korath." He smiles. "But the Quarg have been here for eons. In our time, we have discovered... 'devices,' beyond our ability to scientifically explain. One of these would be of use to you." He pulls out a glass orb, and it floats to the table; there seems to be a black worm with purple hairs inside. The orb has a blinking blue light on top.`

Reviewer 2: If the idea is that this person would be taken to Deep Space 7C, that's not millions of kilometres, that's something like 200000 light years.
Also, is this stuff like what the Remnant have?
Can we... not have this here?
Also, I don't believe that the Quarg would do this. Certainly, not like this.

			choice
				`	(Chat with the Philosopher's Paradise citizens before you go.)`
				`	(It's been a long day. Leave for your ship.)`
					goto leave
			`	You approach the human, Hai, and Korath who seem to represent the people of Philosopher's Paradise. The human turns to you and says, "You look like an asker of questions. What do you want to know?"`
				goto "actual questions"
			label questions
			`	"Do you have more questions?"`
			label "actual questions"
			choice
				`	"Do you want me to take you home?"`
					goto home
				`	"Do you know anything about the microtransceivers?"`
					goto "microtransceivers"
				`	"What can you tell me about this city?"`
					goto place
				`	"What do you do here?"`
					goto do
				`	"What do you know about this area of space?"`
					goto space
				`	"How did the Korath get here?"`
					goto korath
				`	"Can you speak to other Korath for me?"`
					goto "translation services"
				`	"Why were there Korath Raiders in this system?"`
					goto korath
				`	"I have no more questions."`
			`	"It was a pleasure meeting you," says the human leader.`
				goto leave
			label korath

> 			`	The Korath leader begins speak, with the Hai translating. "A few years ago, three of our raiding ships jumped into the Alcyone system. Numerous pirate vessels left Stormhold and disabled all of our ships. A large human ship boarded our flagship while us on the other two vessels listened over communicators. All we heard was screaming. The pirates used deadly gas - what you call 'nerve gas' - to kill the entire crew.  The death is excruciatingly painful to Korath. Not wanting that fate, the rest of the raiders on the second and third Korath ships surrendered. We worked as slaves for the pirate gangs, but now we're here, safe, and free. We are not scientists like the others here, but we help as we can.`

Reviewer 3: ```suggestion
			`	The Korath leader begins speak, with the Hai translating. "Several generations ago, three of our raiding ships jumped into the Alcyone system. Numerous pirate vessels left Stormhold and disabled all of our ships. A large human ship boarded our flagship while us on the other two vessels listened over communicators. All we heard was screaming. The pirates used deadly gas - what you call 'nerve gas' - to kill the entire crew.  The death is excruciatingly painful to Korath. Not wanting that fate, most of the raiders on the second and third Korath ships surrendered. We worked as slaves for the pirate gangs, but now we're here, safe, and free. We are not scientists like the others here, but we help as we can.`
```

Reviewer 4: This may also be a good place to incorporate a reference to the short lifespan/generation time of the Korath?

			`	"The pirates must defend this planet against a space-based lifeform that attacks in flocks. Our ships gave them a significant advantage. Fortunately, we have one more left defend against any threats." The Korath eyes the Republic operatives as he waits for the translator to catch up.`

> 			`	"I must ask you keep the presence of Korath here a secret. If the other Exiles hear of this place, they may attack to retrieve us, or simply to pillage. We will remain here in hiding unless Korath space becomes safe."`

Reviewer 3: ```suggestion
			`	"I must ask you keep the presence of Korath here a secret. If the Archons or other Exiles hear of this place, they may attack to retrieve us, or the Exiles may simply pillage. We will remain here in hiding unless the Exiles finally learn to settle on their planet."`
```

				goto questions
			label "translation services"
			`	The Korath leader hisses and replies while the Hai translates. "No, we cannot. Few among the Korath know your language, and we must remain here in secret. Perhaps in time there will be a way, but not yet."`
				goto questions
			label "microtransceivers"
			`	The Korath leader looks to the Hai, who responds, "Is that how you were led back here?"`
			`	"Yes," you reply.`

> 			`	The Hai leader continues, "Excellent! That was the plan. Our overlords ordered us to create a means of tracking their prey. The microtransceivers were a combination of Hai medical microbots and what our Korath friends here could tell us. None of the Korath were scientists, but the principles of their science are part of their engineers' training. With their knowledge, we augmented the transceivers to transmit in hyperspace as well, hoping it would lead someone back here."`

Reviewer 3: ```suggestion
			`	The Hai leader continues, "Excellent! That was the plan. Our overlords ordered us to create a means of tracking their prey. The microtransceivers were a combination of Hai medical microbots and what our Korath friends here could tell us. None of the surviving Korath were scientists, but many of them seemed to have an uncanny knack for engineering. With their help, we augmented the transceivers to transmit in hyperspace as well, hoping it would lead someone back here."`
```

			`	The human leader adds, "I can scarcely believe it worked, but I'm very glad it did."`
				goto questions
			label home
			`	The three leaders laugh. The Hai answers, "We are home. This is a place of learning, where three races join as one to seek knowledge. Some recent kidnapping victims wish to leave. We will help you find them and send them home."`

> 			`	The Korath speaks next, and the Hai translates. "Where my people come from, everyone is starving, barely subsisting. We have to raid for supplies, and those of us who refuse to fight are treated poorly. This is a much better place for us."`

Reviewer 3: ```suggestion
			`	The Korath speaks next, and the Hai translates. "Where my people came from, everyone was starving, barely subsisting. We had to raid for supplies, or slave away in bleak factory stations. This is a much better place for us."`
```

				goto "actual questions"
			label questions
			`	"Have you more questions?"`
			label "actual questions"
			choice
				`	"Can the Quarg help us more directly, with ships and troops?"`
					goto help
				`	"How does this device work?"`
					goto work
				`	"I'm ready to go."`
					goto accept
			label help
			`	"The Quarg do not venture near that region of space. It is filled with... uncomfortable history. We do not discuss that matter with outsiders."`
				goto questions
			label work
			`	"The Quarg do not know all things. Some secrets of the universe are far beyond us. Only the creators of the orb know all its secrets."`
				goto questions
			label accept
			`	As you answer, Stephen enters the room. "Tutitot has agreed to help. We have our spy."`
			`	The Quarg nods and rises, handing you the orb. "I wish you success, wise humans and Hai. Your people are led well under your guidance." With a bow, Quarg leaves the room.`
			`	Stephen Moray rises as well and tells you, "When you're ready, come to the spaceport." He hands you an access card with a room number. "You'll need room for one passenger," he explains, and then he follows the Quarg out.`

> 			`	Teeneep thanks you with <payment>. "I cannot understate how much you've helped us. On behalf of the Hai, thank you. Now, let us put this plan into action."`

Reviewer 5: ```suggestion
			`	Teeneep thanks you with <payment>. "I cannot overstate how much you've helped us. On behalf of the Hai, thank you. Now, let us put this plan into action."`
```

			`	Teeneep explains her "intentional kidnapping" plan to the others, and Stephen says, "Believe it or not, I met a Hai recently that might be able to help. His name is Tutitot Eeegin, and he's expressed interest in helping us solve this problem. He's clever, and he has some experience acting." Stephen excuses himself to contact Tutitot while the Quarg turns to you and Teeneep.`
			`	"Teeneep asked if the Quarg have something you could use that would let Tutitot send telepathic signals millions of your kilometers. Alas, the Quarg have been unable to create such technology, nor have the Korath." He smiles. "But the Quarg have been here for eons. In our time, we have discovered... 'devices,' beyond our ability to scientifically explain. One of these would be of use to you." He pulls out a glass orb, and it floats to the table; there seems to be a black worm with purple hairs inside. The orb has a blinking blue light on top.`

> 			`	"This worm will let Tutitot talk to the holder of this orb regardless of distance. Bring Tutitot to the kidnappers. Go then to Devil-Eye to place the orb. When Tutitot has reached the Philosopher's Paradise, it will send you a... feeling. You will know it's time."`

Author: Ah this and the next two are all legacies of the recent rework of it to accommodate for certain other plans having changed in the two years since it was first PR'd. Good catches.

> 			`	"This worm will let Tutitot talk to the holder of this orb regardless of distance. Bring Tutitot to the kidnappers. Go then to Devil-Eye to place the orb. When Tutitot has reached the Philosopher's Paradise, it will send you a... feeling. You will know it's time."`

Reviewer 4: Makes me feel a lot better about the confusion I was having, then :)

				goto "actual questions"
			label questions
			`	"Have you more questions?"`
			label "actual questions"
			choice
				`	"Can the Quarg help us more directly, with ships and troops?"`
					goto help
				`	"How does this device work?"`
					goto work
				`	"I'm ready to go."`
					goto accept
			label help
			`	"The Quarg do not venture near that region of space. It is filled with... uncomfortable history. We do not discuss that matter with outsiders."`
				goto questions
			label work

> 			`	"The Quarg do not know all things. Some secrets of the universe are far beyond us. Only the creators of the orb know all its secrets."`

Author: So I've now given you a pictographic instruction card when the orb is handed over, with the implication that Tutitot was briefed more fully and you have the instructional reference. You still find out how it works as you go but I think it should be more logical. (When the next commit occurs.)

			`	"They're waiting on <destination> for you to pick them up. In total, there are 200 operatives and 70 tons of gear that you'll be responsible for transporting. The Republic will give you your usual payment of <payment>. I'll keep the Unfettered distracted - and secret - until you return."`
				accept
	on complete
		payment 600000

mission "Hai Rescue [C06] Meet the Fleet"
	landing
	name "Rejoin the Fleet at <planet>"
	description "Rejoin the Unfettered at <destination>. Payment is <payment>."
	source "Valhalla"
	destination "Foundry"
	passengers 200
	cargo "anti-pirate gear" 70
	blocked "You don't have the room for all <bunks> operatives and their <tons> of equipment. Land here again once you have that much space."
	to offer
		has "Hai Rescue [C05] Operatives: done"
	on visit

> 		dialog `You've fogotten some of the operatives or their equipment; they're probably in your escorts. Wait for the escorts to arrive, and land again to finish this mission.`

Author: ```suggestion
		dialog `You've forgotten some of the operatives or their equipment; they're probably in your escorts. Wait for the escorts to arrive, and land again to finish this mission.`
```

			choice
				`	(Chat with the Philosopher's Paradise citizens before you go.)`
				`	(It's been a long day. Leave for your ship.)`
					goto leave
			`	You approach the human, Hai, and Korath who seem to represent the people of Philosopher's Paradise. The human turns to you and says, "You look like an asker of questions. What do you want to know?"`
				goto "actual questions"
			label questions
			`	"Do you have more questions?"`
			label "actual questions"
			choice
				`	"Do you want me to take you home?"`
					goto home
				`	"Do you know anything about the microtransceivers?"`
					goto "microtransceivers"
				`	"What can you tell me about this city?"`
					goto place
				`	"What do you do here?"`
					goto do
				`	"What do you know about this area of space?"`
					goto space
				`	"How did the Korath get here?"`
					goto korath
				`	"Can you speak to other Korath for me?"`
					goto "translation services"
				`	"Why were there Korath Raiders in this system?"`
					goto korath
				`	"I have no more questions."`
			`	"It was a pleasure meeting you," says the human leader.`
				goto leave
			label korath

> 			`	The Korath leader begins speak, with the Hai translating. "A few years ago, three of our raiding ships jumped into the Alcyone system. Numerous pirate vessels left Stormhold and disabled all of our ships. A large human ship boarded our flagship while us on the other two vessels listened over communicators. All we heard was screaming. The pirates used deadly gas - what you call 'nerve gas' - to kill the entire crew.  The death is excruciatingly painful to Korath. Not wanting that fate, the rest of the raiders on the second and third Korath ships surrendered. We worked as slaves for the pirate gangs, but now we're here, safe, and free. We are not scientists like the others here, but we help as we can.`

Author: This is meant to be an interpretation issue, indicating less than a fully educated use of language.

			`	The human leader adds, "I can scarcely believe it worked, but I'm very glad it did."`
				goto questions
			label home
			`	The three leaders laugh. The Hai answers, "We are home. This is a place of learning, where three races join as one to seek knowledge. Some recent kidnapping victims wish to leave. We will help you find them and send them home."`
			`	The Korath speaks next, and the Hai translates. "Where my people come from, everyone is starving, barely subsisting. We have to raid for supplies, and those of us who refuse to fight are treated poorly. This is a much better place for us."`
			`	The human leader concludes, "Most of the humans in this city have been here for several generations. This has always been home to us."`
				goto questions
			label place

> 			`	"We understand you have heard some of the story from your spy," the human replies, "but I am happy to fill in the details. This place was founded over a hundred twenty years ago by anarchists from the colony of Freedom. With the Syndicate on their doorstep, some of them evolved into anarcho-syndicalists and attempted to bring it down from within. To do this, they allied with less peaceful anarchists and pirates. Those less savory types kidnapped researchers and engineers, forcing them to devise new weapons or medicines, and who eventually inherited control here after the syndicalists were wiped out. Our city also has manufacturing facilities where we created goods the pirates could not easily build in human space, where their facilities are destroyed regularly.`

Author: This one is also intentional specificity of speech pattern.

			`	"There have been various attempts to take the city by force or by money. Early on, there were attempts by Alpha groups, and then various illegal organizations who tracked this place down from rumors about their rivals. There was even someone from the Syndicate who came with his personal fleet, though none of them returned home from that attempt. Some used chemical or biological weapons, others sent ships. Many offered money or favors. Philosopher's Paradise has always been an independent country, and always will be.`

> 			`	"Now that the pirates are gone, us civilians may rule without fear, and for that you have our gratitude. As the upper region of the city has already been destroyed, we have little to fear from attack by space creatures in the short-term, and soon we'll have our shipyards at full capacity such that we may defend ourselves.`

Author: And again

> 			`	"Now that the pirates are gone, us civilians may rule without fear, and for that you have our gratitude. As the upper region of the city has already been destroyed, we have little to fear from attack by space creatures in the short-term, and soon we'll have our shipyards at full capacity such that we may defend ourselves.`

Reviewer 4: I did think these couple of mistakes seemed out of place in your writing, but not strongly enough to pick up them as specific speech patterns or translation issues. No specific suggestions on how to make it clearer, just something to maybe keep in mind?

			`	"They moved their Jump Drive and Fuel Processor to a larger vessel and padded it with armor from the crashed Korath ship. They passed through the Whispering Void and made a colony on the Philosopher's Paradise. It has grown over the years, as they build the facility out and down and capture more researchers."`
				goto questions
			label large
			`	"I have not explored all of it. The top five levels are docking bays and cargo storage. The next twenty levels have people, laboratories, factories, shipyards, and communal areas. The twenty levels below that have industrial machinery, generators, and life support systems. I hear rumors there are lower levels, but I know not what is there.`
			`	"As for population, there are at least 80,000 human civilians, 20,000 Hai, and about 300 Korath Exiles. There are also at least 5,000 pirates, all human."`
				goto questions
			label defenses
			`	"The pirates regularly capture or buy capital ships and bring them here. I think they may have even built some here. Their fleet has grown to dozens - they use them for mining asteroids because they have so many.`

> 			`	"The fleet never leaves; it defends the pirates against terrible threats. Some kind of flocking apex predator attacks the colony frequently. The creatures cannot land on the moon, but they do bombard the surface, damaging the upper parts of the city There are many automated turrets and thousands of well-armed pirates in the city. You wouldn't be attacking a base. You would be attacking a well-defended colony."`

Reviewer 6: missing a . after city

	description "Meet Teeneep on <destination>."
	source "Darkwaste"
	destination "Hai-home"
	clearance
	to offer
		has "Hai Rescue [A09] Slaver Ships: done"
	on offer
		event "hai and human ships at wormhole"
		conversation
			`You land outside the hidden base and you notice the strangely-outfitted Centipede is gone. It has been replaced with a pair of Lightning Bugs which appear to have standard outfits. You walk into the base and see some researchers you saw before, but Teeneep is absent.`
			`	You explain your discoveries and the Hai look increasingly concerned with each sentence you speak.`
			`	After paying you, they explain, "Not long after you left, we went public with our story. We told everyone who would listen. Then the government received your findings through Earth authorities. There's been a big outpouring of support, and a lot of anger at the Elders for letting this go on for so long."`
			`	"Two days ago, there was an election. A third of the Elders have been replaced, which is the fastest change we've had since the Korath invasion began. Teeneep is being inaugurated today, as the youngest 'elder' since the Elder Council was formed. I fear a time of chaos has begun."`

> 			`	 "We'll contact her by hyperwave and tell her everything you told us. Please fly immediately to <destination>. The Elders will no doubt meet as soon as you arrive."`

Author: I kinda want to say it should be "We'll send her a hyperwave" or some such thing. There's probably an in-universe for the type of message, so it doesn't hurt to name it. I agree existing implementation was clunky though.

> 			`You land outside the hidden base and you notice the strangely-outfitted Centipede is gone. It has been replaced with a pair of Lightning Bugs which appear to have standard outfits. You walk into the base and see some researchers you saw before, but Teeneep is absent.`
> 			`	You explain your discoveries and the Hai look increasingly concerned with each sentence you speak.`
> 			`	After paying you, they explain, "Not long after you left, we went public with our story. We told everyone who would listen. Then the government received your findings through Earth authorities. There's been a big outpouring of support, and a lot of anger at the Elders for letting this go on for so long."`
> 			`	"Two days ago, there was an election. A third of the Elders have been replaced, which is the fastest change we've had since the Korath invasion began. Teeneep is being inaugurated today, as the youngest 'elder' since the Elder Council was formed. I fear a time of chaos has begun."`
> 			`	 "We'll contact her by hyperwave and tell her everything you told us. Please fly immediately to <destination>. The Elders will no doubt meet as soon as you arrive."`

Reviewer 7: ```suggestion
			`You land outside the hidden base and notice the strangely-outfitted Centipede is gone. It has been replaced with a pair of Lightning Bugs which appear to have standard outfits. You walk into the base and see some researchers you saw before, but Teeneep is absent. You explain your discoveries, and the Hai look increasingly concerned with each sentence you speak.`
			`	After paying you, they explain, "Not long after you left, we went public with our story. We told everyone who would listen. Then the government received your findings through Earth authorities. There's been a big outpouring of support, and a lot of anger at the Elders for letting this go on for so long.`
			`	"Two days ago, there was an election. A third of the Elders have been replaced, which is the fastest change we've had since the Korath invasion began. Teeneep is being inaugurated today, as the youngest 'elder' since the Elder Council was formed. I fear a time of chaos has begun.`
			`	 "We'll send her a hyperwave and tell her everything you told us. Please fly immediately to <destination>. The Elders will no doubt meet as soon as you arrive."`
```
This one has hyperwave in it. I'll let you decide if you wanna just go with that or workshop it a bit more.

		conversation
			`When you arrive at <origin>, Teeneep meets you at your ship as soon as you touch down. She's wearing the robes of a Hai Elder and has four bodyguards. They're wearing full body armor with personal energy shields. Their pulse rifles are the lethal variety built during the Korath war, rather than the non-lethal ones made today. You notice Teeneep is wearing a backpack with a personal shield.`
			`	She greets you, "Times are changing quickly, <first>. The Navy sent a message ahead of you. Your information is considerably more detailed, but theirs was enough to anger the entire Elder Council. I advised against rash action, but my words went unheeded."`
			`	"The Hai Elders are so angered at the killing of our people that we have officially closed our borders. A large armada guards the wormhole at Waypoint. The human authorities have responded by putting a fleet at Ultima Thule. This situation is explosive and may soon lead to war."`
			`	"Hai are all banned from entering human space now, and humans are not allowed in and out except for critical travel. My new position as an Elder grants me special privileges."`

> 			`	"Please, in the name of peace, escort my ship to meet your authorities. I must also see Waypoint and Ultima Thule for myself. I dare not try this without your escort. You will be well-paid."`

Author: `- each of with full body armor with personal energy shields.` grammar?

> 			`	"Please, in the name of peace, escort my ship to meet your authorities. I must also see Waypoint and Ultima Thule for myself. I dare not try this without your escort. You will be well-paid."`

Reviewer 7: Oops! Fixed.

		not "Hai Rescue [B02B] Diplomacy: Syndicate (Retry): done"
		not "Hai Rescue [B02B] Diplomacy: Syndicate (Retry): active"
	on visit
		dialog `To finish this mission, you must visit Waypoint and Ultima Thule, and then land with Teeneep's Centipede.`
	on enter "Waypoint"
		dialog `There is a massive fleet of Hai ships here, and more coming in all the time. If this fleet were to enter human space, we might be overrun.`
	on enter "Ultima Thule"
		dialog `Although it is reassuring to see humans come together for a common cause, you don't look forward to seeing two species enter an all-out war over a misunderstanding. You also wonder just how effectively the human authorities are keeping the Hai a secret with a huge fleet at an alien station broadcasting Syndicate advertisements...`
	on enter "Mirfak"
		dialog `As you enter the system, authorities ask you to land as soon as possible with Teeneep and meet a Syndicate representative.`
	on offer
		conversation
			`Teeneep meets you at your ship again, "Are you ready to escort me to <destination>?"`
			choice
				`	"Of course. Let's go."`
					accept
				`	"Not yet. I need to make some preparations first."`
					defer
	on complete

> 		# Lower payment because she's annoyed at the delay.

Author: On investigation, I'm not sure this is actually possible. It would require placing a branch _before_ the first line of dialogue (of which there is only one). And I don't believe the game allows that. 
Which means such a variation would require an expansion of the text, which I feel is unnecessary.

			`	A formal delegation meets Teeneep at her ship, but she awaits you before continuing into the building. It takes about a ten minutes of elevators and walkways to get to the top of the building: the Parliament Chamber. The chamber is empty; your delegation is walking through it to a room on the far end of the chamber.`
			choice
				`	(Enter the room.)`
					goto room
				`	(Admire the scenery.)`
			`	The entire outer wall of the 200-meter-wide room is a transparent metal which lets you see the steel forest of buildings that coat Earth. You're at the top of the tallest structure ever built by the human race.  The high altitude of the Parliament Chamber symbolizes its status in society: the supreme elected authority that oversees billions of people.`
			`	Beyond the chamber resides the Crystal Room, perhaps the most famous room in the galaxy. It was there that monumental treaties were signed: the end of the Alpha Wars, the formal declaration of the illegality of slavery, the Syndicate and Deep Sky joining the Republic, the peace treaty with the Free Worlds, and numerous others.`

> 			`	Teeneep looks back at you, waiting for you to rejoin her.`

Reviewer 8: Deep Sky is a corporation headquartered in the Deep, but the region/political entity that joined the Republic seems to be simply the Deep.

Reviewer 9: i'm pretty sure the deep used to be governed by deep sky, so technically deep sky joined the republic.

Reviewer 8: This seems odd. In the timeline on the wiki here, the Deep is described as having been under one megacorporation during the Alpha Wars, but had a coordinated popular uprising result in the establishment of a democratic government in 2550. The in-game description of Huginn Station describes the Deep as the first large region of space to have a democratic government, predating the Republic by nearly two centuries, and Leviathan Station describes the Deep as one of the only regions of space with a democratic government nearly four hundred years ago and says that today it _remains_ under government control, because the citizens of the Deep are wary of any private corporation controlling the industry that makes interstellar travel possible.

Author: That's actually a good catch, I'll put that on the list for the next commit.

				accept

mission "Hai Rescue [B07] Return"
	landing
	name "Missing Researchers"
	description "Land on <destination> to receive report back to Teeneep. Payment is <payment>."
	source "Devil-Eye"
	destination "Sunracer"
	on enter "Mirfak"
		dialog `As you enter the system, authorities on <planet> give you landing coordinates for a secret Syndicate base. They ask you to land as soon as possible.`
	to offer
		has "Hai Rescue [B06] Tutitot Reports: done"
	on offer
		conversation
			`You land and the Drak orb appears in your cockpit, right in front of your face. You don't know how it entered your ship, but this is Drak technology. If Quarg do not understand it, it is far beyond human understanding.`
			`	Tutitot's visage appears in the orb.`
			`	"I am in the Philosopher's Paradise. The city is older and larger than we thought, and there are many researchers. I found some that were kidnapped recently. An alliance of pirate groups rules here, but the other denizens are friendly. There are thousands of well-armed troops and dozens of capital ships."`
			`	"You should return soon to Teeneep to arrange our rescue."`
			label questions
			`	"Is there anything else you wish to know?"`

> 			choice

Author: Wiz actually already rewrote what I think you've been reading and that was PR'd to my branch and merged before.

> 			choice

Author: Ok I think I see what you meant, I have expanded slightly on how he learned each piece of info when he responds.

		conversation
			`You meet Teeneep outside her Centipede and you're surprised to see her with no bodyguards. Instead of her Hai Elder robes, she's wearing what looks like a fencing outfit. There are two swords on her back and several balanced knives on her belt. She's wearing a headband with a medallion on it that looks like an athletic medal.`
			`	She pays you for the trip and says, "These blades are not ceremonial. Leave your gun behind or the Unfettered may think you're an assassin. I'll protect you."`
			`	You hand your gun to a nearby Unfettered, and hope Teeneep is as capable as she claims.`
			`	Now that you see her in this guise, you remember a fencing match advertisement. She looks exactly like the fencing champion a few years ago. Is Teeneep a champion fencer as well as a civil rights leader and Hai Elder? From appearances, she hasn't been in adulthood for long. Though with such celebrity power, it's no surprise she rose in society quickly.`
			`	"Let's go," she says.`
			choice
				`	"What is your plan?"`
				`	(Follow her.)`
					goto follow

> 			`	"To Unfettered Hai leadership, this meeting is all a political ploy. They know it's in their best interest to help us. It will raise public opinion towards them among the Fettered and likely lure more recruits to their side. They cannot be seen as helping a weak coward though. I have to prove myself in front of the troops."`

Reviewer 10: Just a thought, it seems odd that Teeneep would refer to her own people as the Fettered Hai, as it would confirm the unfettered conspiracy theory

Reviewer 10: Upon further reading, scratch this comment

			`	"Well if it isn't <first>, aren't you a sight for sore eyes," he greets you.`
			`	The lack of his usual formality does not go unnoticed, you don't think he's ever failed to call you Captain before.`
			`	"That bad hey?"`
			`	"I've been bombarded by Elders and functionaries constantly asking for explanations and details and communicating idle warnings for the tiniest infringements of ship movements in the vicinity of Mountaintop. Terry's been arrested for trying to leave the station as an unofficial representative, Alondo somehow seems to have made friends but none of them will speak while I'm in the room, and to top it all off I can't even get hold of a decent scotch."`
			`	He flicks his fingers dismissively at what appears to be a cheap imported bottle of bourbon. Despite his complaints it is nevertheless nearly empty.`
			`	"Please, tell me there's something I can do to get this over with quicker somehow."`
			choice
				`	"You're in luck, friend."`

> 			`	You explain to him about the upgrades the Unfettered need to their ships in order to join battle, and the plan to rescue the Hai and end the standoff before anyone does anything stupid.`
> 			`	"And of course," he says, "you're not an official representative of the Republic, so you need someone to authorize and organize things if it's to happen quickly."`
> 			`	"Well have no fear, I can definitely do that!" Within a few short minutes you seem to have lifted a handful of years off of Xilin's worried features.`
> 			`	"I'll sort out the approvals, and I'm sure the Marshal can sort out the logistics. Give me the details and we'll have it all waiting for you on Prime by the time you arrive."`
> 			`	You thank him, and as he seems eager to get that ball rolling you leave him to return to your ship.`

Reviewer 7: ```suggestion
			`	You explain the plan to rescue the Hai and end the standoff before anyone does anything stupid, as well as the upgrades the Unfettered need to their ships in order to join the battle.`
			`	"And of course," he says, "you're not an official representative of the Republic, so you need someone to authorize and organize things if it's to happen quickly. Well, have no fear! I can definitely do that!" Within a few short minutes, you seem to have lifted a handful of years off of Xilin's worn features.`
			`	"I'll sort out the approvals, and I'm sure the Marshal can sort out the logistics. Give me the details and we'll have it all waiting for you on Prime by the time you arrive."`
			`	You thank him, and as he seems eager to get that ball rolling, you leave him to return to your ship.`
```
Usual small corrections and a couple style edits.

mission "Hai Rescue [C02] Escort to Arms Purchase"
	landing
	name "Missing Researchers"
	description "Escort the Unfettered Hai Shield Beetles to <destination>. All must arrive intact. Payment is <payment>."
	source "Allhome"
	destination "Prime"
	to offer
		has "Hai Rescue [C01] Bloody Love Blooms: done"
	on visit
		dialog `To finish this mission, all eleven Unfettered Hai Shield Beetles must land with you.`
	on enter Betelgeuse
		dialog `The large fleet of human ships in this system fly the same flag, "Human Alliance," and appear to have outfits from all factions. The human authorities must be concerned about well-equipped Unfettered Hai Shield Beetles wandering through human space. Hopefully this human camaraderie will outlive the current crisis.`
	on offer
		conversation
			`Teeneep comes to you wearing her fencing outfit, still stained with blood, though you notice there's more blood now, in different places than before. She departs from an eleventh Shield Beetle; you can see weapons sticking out that are not of Hai or human origin. Its thrusters are so hot they're still glowing red.`

> 			`	She says, "I've had a good time with the Unfettered," probably referring to her blood stains, "but now we must advance our plans. We need to reach <destination> immediately. Neither the Fettered Hai nor the Republic want this fleet in their space for long. You'll be paid <payment> when we arrive."`

Reviewer 7: ```suggestion
			`	She says, "I've had a good time with the Unfettered," probably referring to her blood stains, "but now we must advance our plans. We need to reach <destination> immediately. Neither the Fettered Hai nor the Republic want this fleet in their space for long. You'll be paid <payment> when we arrive. And also," she says, gesturing to the Shield Beetle behind her, "we'll be escorting this ship in addition to the ten to be refitted."`
```

				accept
	npc save accompany
		government "Hai (Friendly Unfettered)"
		personality escort disables plunders harvests
		ship "Shield Beetle (Sunbeam Korath Jump)" "Unfettered Flagship"
		fleet 10
			names "hai"
			variant
				"Shield Beetle (Fuel Jump)"
	npc
		government "Human Alliance"
		personality escort heroic staying opportunistic
		system Betelgeuse
		fleet "Human Wormhole Guards" 20
	on complete
		payment 600000
		conversation

> 			`As you come in to land the local Republic officials inform you that the outfitters have emptied ten large bays so nobody notices the Hai on the ships. There are few planets with such a large excess capacity for capital ship outfitter bays; no wonder the Hai picked Prime.`

Reviewer 10: It may make more sense for Xilin to have picked Prime, as he would have a more detailed understanding of human space than the Hai in this context.

> 			`As you come in to land the local Republic officials inform you that the outfitters have emptied ten large bays so nobody notices the Hai on the ships. There are few planets with such a large excess capacity for capital ship outfitter bays; no wonder Xilin picked Prime.`

Reviewer 7: ```suggestion
			`As you come in to land, the local Republic officials inform you that the outfitters have emptied ten large bays so nobody notices the Hai on the ships. There are few planets with such a large excess capacity for capital ship outfitter bays; it's no wonder why Xilin picked Prime.`
```

mission "Hai Rescue [C03] Unfettered Escort"
	landing
	invisible
	source "Prime"
	destination "Foundry"
	to offer
		has "Hai Rescue [C02] Escort to Arms Purchase: done"
	on enter Almaaz

> 		dialog `You receive a message once you are safely to the staging system of Almaaz; the Republic authorities pay you 600,000 credits for your assistance thus far. They imply that if you pull this off, further rewards may await.`

Reviewer 7: ```suggestion
		dialog `You receive a message once you are safely to the staging system of Almaaz: the Republic authorities are paying you 600,000 credits for your assistance thus far. They imply that if you pull this off, further rewards may await.`
```

		payment 600000

# Special outfitting for the Philsopher's Paradise battle.

mission "Hai Rescue [C04] Foundry"
	landing
	name "Missing Researchers"
	description "Escort the combined fleet to <destination>. All Shield Beetles must arrive intact. Payment is <payment>."
	source "Prime"
	destination "Foundry"
	to offer
		has "Hai Rescue [C02] Escort to Arms Purchase: done"
	on visit
		dialog `To finish this mission, all eleven Unfettered Hai Shield Beetles must land with you.`
	on offer

> 		log `Eleven Unfettered Hai Shield Beetles are equipped for the battle at Philosopher's Paradise. A fleet of human capital ships escort them, making sure they don't stray from their intended purpose. Two hundred Republic special operatives, trained in anti-pirate combat, await you at the spaceport at Foundry. Teeneep and General Choot'k plan on leading the Unfettered Fleet personally, from the Unfettered flagship.`

Reviewer 7: Same as above for logs.

	npc save accompany
		government "Hai (Friendly Unfettered)"
		personality escort disables plunders harvests
		ship "Shield Beetle (Sunbeam Korath Jump)" "Unfettered Flagship"
		fleet 5
			names "hai"
			variant
				"Shield Beetle (Particle Jump)"
		fleet 5
			names "hai"
			variant
				"Shield Beetle (Plasma Blaster Jump)"
	npc
		government "Human Alliance"
		personality escort heroic waiting opportunistic
		fleet "Human Wormhole Guards" 20
	on complete
		payment 600000

mission "Hai Rescue [C05] Operatives"
	landing
	name "Pick up Operatives"
	description "Head to <destination> to pick up 200 operatives and their 70 tons of gear. Payment is <payment>."
	source "Foundry"
	destination "Valhalla"
	to offer
		has "Hai Rescue [C04] Foundry: done"
	on offer
		conversation
			`Deputy Director Stephen Moray greets you as you exit your ship. "The Unfettered are fearsome pilots, as are you, but we need some special operatives to storm the city. The Republic has rounded up the best troops in the whole of the Republic and has armed them with some special equipment. They're being very hush-hush about it, but I understand it is advanced anti-pirate gear Deep Sky has been developing."`

> 			`	"Head to <destination> to pick up the 200 operatives and their 70 tons of gear. The Republic will give you your usual payment of <payment>."`
> 			`	"I'll keep the Unfettered distracted... and secret... until you return."`

Reviewer 7: ```suggestion
			`	"Head to <destination> to pick up the 200 operatives and their 70 tons of gear. The Republic will give you your usual payment of <payment>. I'll keep the Unfettered distracted - and secret - until you return."`
```

				accept
	on complete
		payment 600000

mission "Hai Rescue [C06] Meet the Fleet"
	landing
	name "Rejoin the Fleet at <planet>"
	description "Rejoin the Unfettered at <destination>. Payment is <payment>."
	source "Valhalla"
	destination "Foundry"
	passengers 200
	cargo "anti-pirate gear" 70
	blocked "You haven't the room for all <bunks> operatives and their <tons> of equipment. Land here again once you have that much space."
	to offer
		has "Hai Rescue [C05] Operatives: done"
	on visit
		dialog `You forgot some of the operatives or their equipment; they're probably in your escorts. Wait for the escorts to arrive and land again to finish this mission.`
	on offer

> 		log `Agent Kopkow of Republic Intelligence complained that they're running out of cover stories to hide the Hai. He needs the Hai-human conflict resolved without an more public displays. Once I have space for his 200 operatives, I need to pick them up at the Foundry spaceport.`

Reviewer 7: Same as above for logs.

		conversation
			`Soon after landing, you are met by Agent Kopkow, the Republic Intelligence agent you met on Earth. He seems extremely concerned about something.`

> 			`	"It is becoming increasingly difficult to cover up the existence of the Hai. The captains at Ultima Thule already know about the Hai, but we had to tell the crews we were dealing with a major pirate threat. We hired a few pirate gangs, tricking them into flying through the area to back our story. The Shield Beetles at Prime were explained away as experimental ship designs."`
> 			`	"Our cover stories have passed the point of plausibility into the realm of barefaced lying. I hope you can finish this without too many more public displays, otherwise we won't be able to hide the Hai anymore.`

Reviewer 7: ```suggestion
			`	"It is becoming increasingly difficult to cover up the existence of the Hai. The captains at Ultima Thule already know about the Hai, but we had to tell the crews we were dealing with a major pirate threat. We hired a few pirate gangs, tricking them into flying through the area to back our story. The Shield Beetles at Prime were explained away as experimental ship designs. Our cover stories have passed the point of plausibility into the realm of barefaced lying. I hope you can finish this without too many more public displays, otherwise we won't be able to hide the Hai anymore.`
```

> 			`	"I've brought 200 intelligence operatives, specially trained for attacking pirate strongholds, and equipped with advanced weapons that... are on a need-to-know basis. Unfortunately, we possess no ships that can transport that many people safely to the surface. Even a Bactrian would be torn to pieces. You will have to transport the operatives. We can transport them back afterwards."`
> 			`	"Once you're ready for the battle at Philosopher's Paradise, meet our operatives in the spaceport. You will need 200 bunks and 70 tons of cargo space to fit all the operatives and their gear. Please don't take long. We're running out of cover stories."`
> 			`	He walks off dismayed.`

Reviewer 7: ```suggestion
			`	"I've brought 200 intelligence operatives, specially trained for attacking pirate strongholds, and equipped with advanced weapons that... are on a need-to-know basis. Unfortunately, we possess no ships that can transport that many people safely to the surface. Even a Bactrian would be torn to pieces. You will have to transport the operatives. We can transport them back afterwards. Once you're ready for the battle at Philosopher's Paradise, meet our operatives in the spaceport. You will need 200 bunks and 70 tons of cargo space to fit all the operatives and their gear. Please don't take long. We're running out of cover stories." He walks off looking very stressed.`
```

				accept
	on complete
		payment 600000

mission "Hai Rescue [C07A] Devil-Hide Fleet"
	landing
	invisible
	source "Foundry"
	destination "Hai-home"
	to offer
		has "Hai Rescue [C06] Meet the Fleet: done"

> # Workaround for bug #1936 is to introduce another bug the user is unlikely to

Reviewer 7: ```suggestion
# Workaround for bug #1938 is to introduce another bug the user is unlikely to
```
I assume this is in reference to #1938? #1936 has been closed for years.

# trigger. Visit Hai-home with this fleet and it vanishes.
#	to complete
#		has "Hai Reveal [D01] Cleanup: done"
	to fail
		or
			has "Hai Rescue [C07] Devil-Hide Battle: failed"
			has "Hai Rescue [C07] Devil-Hide Battle: declined"
			has "Hai Reveal [D01] Cleanup: failed"
			has "Hai Reveal [D01] Cleanup: declined"
			has "Hai Rescue [C08] Freedom: failed"
			has "Hai Rescue [C08] Freedom: declined"
	npc
		government "Hai (Friendly Unfettered)"
		personality escort heroic
		fleet 5
			variant
				"Shield Beetle (Particle Jump)"
		fleet 5
			variant
				"Shield Beetle (Plasma Blaster Jump)"

> mission "Hai Rescue [C07] Devil-Hide Battle"

Reviewer 7: A question: how does this mission work? It states that the completion criteria is to dominate the planet, but as far as I know there isn't actually a way to make dominating a planet a mission. I'm not code-smart enough to understand how this works after 5 minutes of staring at it.

Author: It works in the sense that there are precisely 0 ways to land without dominating the planet (it cannot be bribed) so you cannot reach the destination of the mission without dominating.

	planet "Philosopher's Paradise"
		tribute 7000
			threshold 3000
			fleet "Devil-Run Warships" 5

mission "Hai Rescue [C08] Freedom"
	landing
	name "Missing Researchers"
	description "Escort General Choot'k's ship to <stopovers> and then <destination>. Payment is <payment>."
	source "Philosopher's Paradise"
	stopover "Foundry"
	destination "Hai-home"
	to offer
		has "Hai Rescue [C07] Devil-Hide Battle: done"
	on visit
		dialog `To finish this mission, you need to wait for General Choot'k's ship to arrive and land again.`
	on fail
		fail "Hai Rescue [C07A] Devil-Hide Fleet"
	on decline
		fail "Hai Rescue [C07A] Devil-Hide Fleet"
	on offer
		fail "Hai Rescue [B04Z] Guard"
		log `Factions` `Philosopher's Paradise` `Formerly a pirate stronghold, this city of scientists is a hundred twenty years old now. Korath, Hai, and Humans live peacefully, working side-by-side, and most have no desire to leave. They value their independence and have refused Republic membership. This city orbits a strange world that defies analysis in a region of space that has never been touched by civilization. Quarg refuse to go here, and Drak consider it outside their domain.`

> 		log `Liberated Philosopher's Paradise. They said they have inventions that may be of use to me, but their facilities are damaged. As they repair the facilities, they'll make new technologies available to me as their friend.`

Reviewer 7: Same as above for logs.

			`	Their guns are able to literally melt away meter-thick walls in seconds. They have gas bombs that release an extremely powerful stun gas which knocks people out for hours. The pirates outnumber the Republic operatives twenty-five to one, but this huge advantage in equipment, combined with training, gives the operatives a fighting chance.`
			`	The operatives move room to room, neutralizing all opposition with ease until the pirates switch tactics. They begin using electromagnetic pulse devices to shut down the suits (and all other electronics nearby). This leaves the operatives helpless for about ten seconds.`
			`	After a bloody struggle, forty-three surviving operatives win the fight, clearing the way to a large blast door. The vast door is twelve meters high and wide, and scanners show it to be six meters deep. The operatives focus all of their suits' firepower on one part of the door, eventually burning a hole through it. They pass through into the darkness of the next room.`
			choice

> 				`	Follow them.`

Reviewer 7: ```suggestion
				`	(Follow them.)`
```

			`	The Hai speaks, "We are the leaders of the civilian government of Philosopher's Paradise. Although we are grateful for your assistance in defeating our violent rulers, we will not accept your dominion either. We know what you did to the Free Worlds. Philosopher's Paradise is an independent nation; we will never join the Republic."`
			`	"I promise our country will turn to peace, and we will send no military ships through the wormhole."`
			`	After several hours of discussions, the operatives leave their suits and surrender, preparing to return peacefully to Republic space.`
			`	Do you want to talk to the Philosopher's Paradise citizens before you go?`
			choice
				`	(Talk to them.)`
				`	(Don't talk to them.)`
					goto leave
			`	You approach what looks like a group of leaders: one human, one Hai, and one Korath. The human turns to you and says, "You look like an asker of questions. How may we enlighten you?"`
				goto "actual questions"
			label questions
			`	"Do you have more questions?"`
			label "actual questions"
			choice
				`	"Do you want me to take you home?"`
					goto home

> 				`	"Do you know anything about the microtranceivers?"`

Reviewer 7: ```suggestion
				`	"Do you know anything about the microtransceivers?"`
```

> 					goto "microtranceivers"

Reviewer 7: ```suggestion
					goto "microtransceivers"
```

				`	"What can you tell me about this city?"`
					goto place
				`	"What do you do here?"`
					goto do
				`	"What do you know about this area of space?"`
					goto space
				`	"How did the Korath get here?"`
					goto korath
				`	"Can you speak to other Korath for me?"`
					goto "translation services"
				`	"Why were there Korath Raiders in this system?"`
					goto korath
				`	"I have no more questions."`
			`	"It was a pleasure meeting you," says the human leader.`
				goto leave
			label korath

> 			`	The Korath leader tells you, "A few years ago, three of our ships jumped into the Alcyone system to raid. Numerous pirate vessels left Stormhold and disabled all of our ships. A large human ship boarded our flagship. All we heard was screaming; the pirates used deadly gas to kill the entire crew."`

Reviewer 7: Hmm. Are Korath even able to phonetically pronounce human languages? Ya know, I have a whole mission chain about a Korath translation device that's currently shut down (maybe not _because_ it's a Korath translation device though).

			`	"You call it, 'nerve gas.' The death is excruciatingly painful to Korath. Not wanting that fate, the crews of the second and third Korath vessels surrendered. Now we're here, safe, and free. We are not scientists like the others here, but we help as we can." The other two leaders smile at him gratefully.`

> 			# Apparently you're not the only one who can pack a Bactrian with
> 			# nerve gas and hundreds of disposable teenage soldiers. The pirates
> 			# disable or destroy Korath Raiders all the time. It seems some are
> 			# adventurous enough to capture the ships.

Reviewer 7: ```suggestion
```

			`	"The pirates must defend this planet against a space-based lifeform that attacks in flocks. Our ships gave them a significant advantage. Fortunately we have one more left, to defend against any threats." He eyes the Republic operatives as he says that.`
			`	The Korath leader continues, "I must ask you hold secret the presence of Korath here. If the other Exiles hear of this place, they may attack to retrieve us, or simply to pillage. We will remain here in hiding until Korath space is safe."`
				goto questions
			label "translation services"
			`	The Korath leader looks worried, "No, we cannot. Few among us Korath know your language, and we must remain here in secret. Perhaps in time there will be a way, but not yet."`
				goto questions

> 			label "microtranceivers"

Reviewer 7: ```suggestion
			label "microtransceivers"
```

			`	The Korath leader looks to the Hai who responds, "Is that how you were led back here?"`
			`	"Yes," you say.`

> 			`	The Hai leader continues, "That is excellent, that was the plan. Our overlords ordered us to make microtranceivers to track their prey. The microtranceivers were a combination of Hai medical microbots and what our Korath friends here could tell us. None of the Korath were scientists, but the principles of their science are part of their engineers' training. With their knowledge, we augmented the transceivers to transmit in hyperspace as well, hoping it would lead someone back here."`

Reviewer 7: ```suggestion
			`	The Hai leader continues, "That is excellent, that was the plan. Our overlords ordered us to make microtransceivers to track their prey. The microtransceivers were a combination of Hai medical microbots and what our Korath friends here could tell us. None of the Korath were scientists, but the principles of their science are part of their engineers' training. With their knowledge, we augmented the transceivers to transmit in hyperspace as well, hoping it would lead someone back here."`
```

				"Rainmaker (Mark II Old Weapons)" 8
	npc
		government "Syndicate"
		system "Mirfak"
		personality heroic staying
		fleet "Large Syndicate" 8
	npc
		government "Syndicate"
		system "Zaurak"
		personality heroic staying
		fleet "Large Syndicate" 8

# The fact that you have to actively go to the spaceport to proc this mission is intentional. It allows the player to take a pause to reorganize themselves if they need to, B01 stays active as a reminder.
# The fight in this mission is very large, but it's also a complete roflstomp. The pirates are outnumbered 2-1, and out-gunned significantly worse than that.
# You as the player have one job: make sure everyone jumps in together, and don't die. During the first campaign it was really important the fights were all balanced for you to be able to tip the balance.
# THIS mission is designed to give you a spectacle. A little visual demonstration of what a real battle could potentially look and feel like with low stakes for you. It's a visual feast, not a challenge.
mission "Hai Reveal [B02] Join the Fleet"
	name "Join the Fleet"
	description "Join the fleet in <waypoint> and move on Stormhold."
	source "Sunracer"

> 	waypoint "Sumar"

Reviewer 11: Since the NPCs always travel to the player, forcing a "rendezvous point" doesn't make much sense. The player could just wait for the ships to come to them and never have to go to Sumar.

Author: The narrative purpose is so that the Hai fleet doesn't fly into places it's not meant to.

Reviewer 11: Well that can't be enforced regardless, so I'd simply remove the waypoint.

Author: No, but with the waypoint, if the player takes the fleet elsewhere they know that they're consciously departing from the narrative.

Reviewer 11: Just toss it.

				accept
	npc
		government "Syndicate"
		personality escort coward opportunistic vindictive
		fleet
			names "syndicate capital"
			variant
				"Protector" 2
				"Vanguard" 3
				"Vanguard (Particle)" 3
				"Splinter (Flamethrower Blaster)" 4
		# This special security force has acquired outfits in the post-war condition that were surplus or black market to upgrade their Splinters.
		# However, they haven't had the time to refit and test out new options for their Protectors or Vanguards.

	npc accompany
		government "Republic"
		system "Ultima Thule"
		personality escort heroic opportunistic waiting
		fleet "Human Wormhole Guards" 20
	npc accompany
		government "Hai"
		system "Ultima Thule"
		personality escort heroic opportunistic waiting
		fleet "Hai Wormhole Guards" 20
	npc accompany
		government "Hai (Friendly Unfettered)"
		system "Ultima Thule"
		personality escort heroic opportunistic vindictive waiting

> 		fleet "Large Friendly Unfettered" 7

Reviewer 11: I believe `accompany` is implicitly `save`, so any of these ships dying fails the mission.

Author: Well that's not what the wiki says, and is very limiting if so.

Reviewer 2: What the wiki says: `accompany: You can only complete the mission if all members of this NPC are in the same system as you.`

Strictly, yes, this is inaccurate. The ships are required to not be disabled, which this doesn't mention, but a ship can't be in the same system as you if it's dead, even if it died in that system.

Author: That seems rather inconvenient if you just want to make sure all *surviving* ships of a certain set are present, but don't actually require all of them to be alive.

Reviewer 2: I'm not sure what the use would be. Are you actually handling it differently if, for example, all the ships are lost?

Author: It's more about not *requiring* a separate mission to track a fleet of NPC's which are there to help and don't necessarily all need to survive.

# spoils of wars against Korath and Wanderers:
ship "Shield Beetle" "Shield Beetle (Sunbeam Korath Jump)"
	outfits
		"Large Heat Shunt" 2
		"Jump Drive"
		"Systems Core (Large)"
		"Quantum Keystone"
		"Outfits Expansion" 3
		"Boulder Reactor" 3
		`"Bufaer" Atomic Steering`
		Sunbeam 6
		"Korath Warder"
		"Small Heat Shunt"
		"Thruster (Planetary Class)"
		"Fuel Processor" 2

# Ambassadorial Centipede designed to fly deep into hostile space:
ship "Centipede" "Centipede (Teeneep)"
	# Add some ambassadorial staff:
	add attributes
		"required crew" 50
	outfits
		"Catalytic Ramscoop"
		"Scram Drive"
		"Chameleon Anti-Missile" 3
		"Quantum Keystone"
		"Outfits Expansion"
		`"Biroo" Atomic Steering`
		"Hai Diamond Regenerator"
		"Hai Williwaw Cooling" 2
		"Hai Chasm Batteries" 2
		"Boulder Reactor"
		Brig
		"Fuel Pod" 3
		"A370 Atomic Thruster"
		"Pulse Rifle" 131

# These ship designs are chosen to reduce CPU load while still keeping
# them visibly usable as anti-pirate and anti-Hai vessels. They won't
# be used in real fights; they're just scenery to make a point about
# human cooperation The mix of faction outfits is because all human
# factions (except remnant) are working together.

> ship "Corvette" "Corvette (Flamethrower Electron Anti-Missile)"

Reviewer 12: I'll move this in the PR I'll send you in a few days.

			`	She re-sheaths her swords and calls you over, "I have brought with me one of the greatest war heroes of the humans."`
			choice
				`	(Walk to her.)`
			`	"Captain <first> <last> led great fleets during a civil war of the humans. I have seen his valor in battle as he's escorted my ship through pirate space. We will have the honor of fighting alongside him."`
			`	The Unfettered leader turns to you and says, "I look forward to seeing you fight. We cannot spare a large fleet though, so I will need your people to upgrade ten Shield Beetles to our specifications. We'll return your outfits after the battle." He motions to another officer who hands him a data card, "These are the instructions."`
			`	Teeneep smiles at you, "Return to Hai-home and inform the Elders that an Unfettered fleet will arrive. We must then arrange for Prime to outfit our ships. When all is ready I and the Unfettered ships will rendezvous with you at Allhome."`
			`	You return to your ship and prepare to leave.`
				accept
	on stopover

> 		dialog `The Hai Elders are surprised and disturbed when they hear your account of what happened. They give you detailed instructions for outfitting the Shield Beetles, and suggested security precautions. They also give you a purchase order with an account number for the outfitters to charge.`

Author: I sort of wanted to separate those two events, I think actually I might move the conversation to the 'on offer' of the mission that proceeds from it.

			`	"There have been various attempts to take the city by force or by money. Early on there were attempts by Alpha groups, and then various illegal organizations who tracked this place down from rumours about their rivals. There was even someone from the Syndicate who came with his personal fleet, though none of them returned home from that attempt. Some used chemical or biological weapons, others sent ships. Many offered money or favors. Philosopher's Paradise has always been an independent country, and always will be."`
			`	"Now that the pirates are gone, us civilians may rule without fear, and for that you have our gratitude. As the upper region of the city has already been destroyed, we have little to fear from attack by space creatures in the short-term, and soon we'll have our shipyards at full capacity such that we may defend ourselves."`

> 			`	"We may never join the Republic or any other government beyond the wormhole but you personally will always be welcome, along with those you trust, for what you have done today. Our research and manufacturing facilities are damaged now, but as they are repaired we will make them available to you."`

Author: In fairness a chunk of this is from your stuff, but I don't think the philosopher's are the ones to explain that. It's implied that they're anti-whatever form of capitalism the Syndicate is, and I go into it in a bit more detail when you eventually hunt down the ringleader at the end.

> 			`	"We may never join the Republic or any other government beyond the wormhole but you personally will always be welcome, along with those you trust, for what you have done today. Our research and manufacturing facilities are damaged now, but as they are repaired we will make them available to you."`

Reviewer 12: I suggest it be split in two parts:

1. The Philosophers would understand their origins and the history of what went on inside
2. The Syndicate bunch would know more about the attempts to take over the place.

You could even have the two groups stay in touch closely all the way to the end. The outside one acts as spies to make sure their brethren are not discovered or defeated.

> 			`	"We may never join the Republic or any other government beyond the wormhole but you personally will always be welcome, along with those you trust, for what you have done today. Our research and manufacturing facilities are damaged now, but as they are repaired we will make them available to you."`

Reviewer 12: If you want to take a pass at this, please do so. Otherwise I'll work on it after I fix the mission coding bugs.

> 			`	"We may never join the Republic or any other government beyond the wormhole but you personally will always be welcome, along with those you trust, for what you have done today. Our research and manufacturing facilities are damaged now, but as they are repaired we will make them available to you."`

Author: The implication of this part is that the Philosophers believe the Syndicalists were fully wiped out, or at the very least those who survived threw their lot in with the pirates. This is deliberate, by the time the events of now are happening the syndicalists are meant to be just a handful of people operating their own secret mission with just Terry on the outside. This is one of the reasons she's so emotional at the end. With the defeat of the pirates she knows that those few left that she'd worked with have also died.

				goto questions
			label leave
			`	As you return to your ship, Teeneep interrupts you. "I need you to escort me one last time, Captain <last>. I must return to <destination> to resolve things with now that the threat is ended. You will be well-paid, of course."`
			`	Anticipating your next question, she says, "The ships of the True Hai will be escorted back with us since the Philosophers are convinced they are at no risk, and still have a ship at their disposal. Some of the True Hai have chosen to remain here and join the philosophers. More than enough bunks have subsequently been freed to return those who have chosen to come home with us. We will have to transport the surviving operatives back to Foundry, the Republic should be able to look after them again from there."`
				accept
	on stopover
		dialog "The operatives depart your ship into the arms of the human teams waiting there to see to their health and collect the dead."
	on complete
		payment 1000000
		fail "Hai Rescue [B01A] Permanent Fleets"
	npc save accompany
		government "Hai (Friendly Unfettered)"
		personality escort heroic

> 		ship "Shield Beetle (Sunbeam Korath Jump)" "Unfettered Flagship"

Author: This is your variant my dude, go nuts. :)

> 		ship "Shield Beetle (Sunbeam Korath Jump)" "Unfettered Flagship"

Reviewer 12: I'll fix this in the PR I'll give you in a few days.

	npc
		government "Hai (Friendly Unfettered)"
		personality escort heroic
		fleet 3
			variant
				"Shield Beetle (Particle Jump)"
		fleet 3
			variant

> 				"Shield Beetle (Plasma Blaster Jump)"

Author: What are the odds that >4 out of 10 ships were destroyed? I sort of picked a number I thought was fairly reasonable, figuring a couple were destroyed, and if some Hai stayed behind they might not have been able to bring every ship back with them.

> 				"Shield Beetle (Plasma Blaster Jump)"

Reviewer 12: If the player is starting this before they left Hai space, then it is likely they'll rely entirely on the Shield Beetles for combat. I tuned the battle so the Shield Beetles are unlikely to win on their own.

> 				"Shield Beetle (Plasma Blaster Jump)"

Author: An earlier mission will have forced the player to have escorts, if only briefly, That's meant to be a pretty big hint... is there a way we can track (without making a mission for every ship) how many were destroyed? Or should I just revisit this and see what can be changed? It seems like it would cause a lot of political trouble if you just left a bunch of unfettered ships with jump drives there unsupervised and told the republic they'd 'find their own way home' unescorted.

> 				"Shield Beetle (Plasma Blaster Jump)"

Reviewer 12: I have a few possible solutions in mind. I'm still working my way through bugs in part 1. I'll try my idea when I get to that point. Hopefully in another 10 hours or so.

	on offer
		conversation
			`You reach the spaceport with Teeneep and head to the room you were told to visit. A Hai stands before you, nervously eyeing the strange orb you're holding. "You must be Tutitot," Teeneep says. The Hai nods. "This is Captain <first> <last>. While you two work together, I will stay behind to prepare for our next steps."`

> 			`	Tutitot addresses you. "The Quarg told me I'm supposed to put the worm in my ear. It's just so... weird and gross." You consult the instruction card and only now realize exactly what that step was. The pair of you look at that step and share a look before you give him the orb. Cautiously he lifts it up and brings it near his head. The worm immediately jumps out and vanishes into his ear. The Hai stiffens for a few seconds, and then returns to normal. "I can... feel the orb in my mind. It's as if it is expectant, waiting, but it seems to see the world around it; in fact it's like the whole world is in my mind! This is amazing!"`

Reviewer 7: ```suggestion
			`	Tutitot addresses you. "The Quarg told me I'm supposed to put the worm in my ear. It's just so... weird and gross." You consult the instruction card and only now realize exactly what that step was. The pair of you look at that step and share a look before you give him the orb. Cautiously, he lifts it up and brings it near his head. The worm immediately jumps out and vanishes into his ear. The Hai stiffens for a few seconds, and then returns to normal. "I can... feel the orb in my mind. It's as if it's expectant, waiting, but it seems to see the world around it; in fact, it's like the whole world is in my mind! This is amazing!"`
```

Author: I find that in speech people typically don't say "it's" twice so close to each other and will avoid the contraction on the second use, especially when trying to explain something. The exception being the particular phrasing of `"It's like it's"`, so I'd rather either keep it separate or switch over entirely.

Reviewer 7: Fair point. I personally prefer `It's like it's` over keeping as-is, but up to you:
```suggestion
			`	Tutitot addresses you. "The Quarg told me I'm supposed to put the worm in my ear. It's just so... weird and gross." You consult the instruction card and only now realize exactly what that step was. The pair of you look at that step and share a look before you give him the orb. Cautiously he lifts it up and brings it near his head. The worm immediately jumps out and vanishes into his ear. The Hai stiffens for a few seconds, and then returns to normal. "I can... feel the orb in my mind. It's like it's expectant, waiting, but it seems to see the world around it; in fact, it's like the whole world is in my mind! This is amazing!"`
```
```suggestion
			`	Tutitot addresses you. "The Quarg told me I'm supposed to put the worm in my ear. It's just so... weird and gross." You consult the instruction card and only now realize exactly what that step was. The pair of you look at that step and share a look before you give him the orb. Cautiously he lifts it up and brings it near his head. The worm immediately jumps out and vanishes into his ear. The Hai stiffens for a few seconds, and then returns to normal. "I can... feel the orb in my mind. It's as if it is expectant, waiting, but it seems to see the world around it; in fact, it's like the whole world is in my mind! This is amazing!"`
```

			`	Now you need to wait for the orb to "send you a feeling" so you know to return here. Tutitot said it would be two or three weeks.`
		log `Dropped Tutitot off to be "kidnapped" and taken to Philosopher's Paradise.`
	on stopover

> 		dialog `You don a protective space suit and leave your ship with the strange orb. As your suit starts to give warnings that it is beginning to register atmospheric drag, the orb flashes and the blinking blue light swirls through it and into your gloved hand causing a tingling sensation to run from your palm to up between your eyes. The now dark orb flies out of your hand to a distance of about fifty meters, hovers there for a few seconds and then disappears. You can only surmise that it must have some sort of cloaking device. With the orb delivered you jet quickly back to your ship before the gravity well can drag you down, it's time to take Tutitot to <destination> to be "kidnapped."`

Reviewer 7: ```suggestion
		dialog `You don a protective space suit and leave your ship with the strange orb. As your suit starts to give warnings that it is beginning to register atmospheric drag, the orb flashes, and the blinking blue light swirls through it and into your gloved hand, causing a tingling sensation to run from your palm to up between your eyes. The now dark orb flies out of your hand to a distance of about fifty meters, hovers there for a few seconds, and then disappears. You can only surmise that it must have some sort of cloaking device. With the orb delivered, you jet quickly back to your ship before the gravity well can drag you down. It's time to take Tutitot to <destination> to be "kidnapped."`
```

			`	"The people have an intense dislike of Alphas, so they've definitely shown up at some point. The pirates say Alphas will take over any organization they join. They think that aliens would do the same, so Devil-Run pirates are only humans."`
				goto questions
			label old

> 			`	"Our intel believed the city was established fifty years ago when Base Principles and their partners were supposed to have been scoured from Syndicate space, but I'm free to access large parts of this place so I've had a good chance to talk to some of the locals. They claim that for the most part they're descendants of anarchists from Freedom. There's also no way a city this size could have been built in only fifty years; it must be well over that.`

Reviewer 7: ```suggestion
			`	"Our intel believed the city was established fifty years ago when Base Principles and their partners were supposed to have been scoured from Syndicate space, but I'm free to access large parts of this place, so I've had a good chance to talk to some of the locals. They claim that for the most part they're descendants of anarchists from Freedom. There's also no way a city this size could have been built in only fifty years; it must be well over that.`
```

Author: I question this one.

Reviewer 7: Two independent clauses, need a coordinating conjunction (`so`) _and_ a comma. The alternative would be separating the two sentences entirely:
```suggestion
			`	"Our intel believed the city was established fifty years ago when Base Principles and their partners were supposed to have been scoured from Syndicate space, but I'm free to access large parts of this place. I've had a good chance to talk to some of the locals. They claim that for the most part they're descendants of anarchists from Freedom. There's also no way a city this size could have been built in only fifty years; it must be well over that.`
```
I guess there's an argument that when spoken aloud, it wouldn't have the "pausing" of a comma/period, but then the disadvantage would be having a very long sentence without many breaks: `Our intel believed the city was established fifty years ago when Base Principles and their partners were supposed to have been scoured from Syndicate space, but I'm free to access large parts of this place so I've had a good chance to talk to some of the locals.`
Ultimately up to you.

Author: What if we did the long sentence and then interrupted the paragraph with a "he pauses for breath" type deal?
I think something like that would also add to the vibe that he's trying to communicate as much as possible in a short window of time.

			`	goto questions`
			label large

> 			`	"I've only seen small parts of it so far but I have seen a map. The top few levels are docking bays and cargo storage, and I can access the next several which house people, laboratories, and communal areas. In parallel with those are factories and shipyards which are closely supervised and lastly, below all of that, there are several levels of industrial machinery, generators, and life support systems.`

Reviewer 7: ```suggestion
			`	"I've only seen small parts of it so far, but I have seen a map. The top few levels are docking bays and cargo storage, and I can access the next several which house people, laboratories, and communal areas. In parallel with those are factories and shipyards which are closely supervised. Lastly, below all of that, there are several levels of industrial machinery, generators, and life support systems.`
```

			`	You explain the plan to rescue the Hai and end the standoff before anyone does anything stupid, as well as the upgrades the Unfettered need to their ships in order to join the battle.`
			`	"And of course," he says, "you're not an official representative of the Republic, so you need someone to authorize and organize things if it's to happen quickly. Well, have no fear! I can definitely do that." Within a few short minutes, you seem to have lifted a handful of years off of Xilin's worn features.`
			`	"I'll sort out the approvals, and I'm sure the Marshal can figure out the logistics. Give me the details, and we'll have it all waiting for you on Prime by the time you arrive."`

> 			`	You thank him, and, as he seems eager to get that ball rolling, you leave him to return to your ship.`

Reviewer 7: ```suggestion
			`	As he seems eager to get that ball rolling, you thank him and leave to return to your ship.`
```
